library/src/enabled/java/com/telefonica/tweaks/Tweaks.kt

```diff
@@ -94,14 +96,18 @@ open class Tweaks : TweaksContract {
 @Composable
 fun NavController.navigateToTweaksOnShake() {
```

Comment:
This feature depends on a NavController, although in some cases we might not have a NavController.

In B2P we have this launched effect that does not depend on it, and also allows a custom action to launch the tweaks:

https://github.com/Telefonica/b2p-germany-android/blob/4a9f90436777833358e891f4864d75ad9308309e/app/src/debug/java/com/telefonica/germany/b2p/core/tweaks/ShakeDetectorWrapper.kt#L18

I think this approach would be very helpful to open tweaks from the command line into a different activity (same as SmartWifi does)

Reply:
I've created another function which does not depends on navcontroller

Reply:
Cool! Thanks!